client/src/components/Island/index.tsx

```diff
@@ -85,18 +84,17 @@ const Island: React.FC = (props: any) => {
   )[0].name;
 
   return (
-
-    <IslandWithFileName width={fileSizeAdjustedWidth}>
-      <IslandContainer
-        minWidth={fileSizeAdjustedWidth}
-        x={fileAnalysis.topLeftCorner.x}
-        y={fileAnalysis.topLeftCorner.y}
-      >
+    <IslandContainer
```

Comment:
Instead of flipping IslandWithFileName and IslandContainer, I think the IslandWithFileName should get the x and y positioning